### DISCUSSION TITLE
Swap the "<space>f" and <space>F" bindings

### DISCUSSION THREAD

**Discussion by leyloe** [helix-editor/helix#11016] [Ideas]
Swap "<space>f" and <space>F" because lower f pops up all files, and upper f pops up current working directory (that will be used the most).

Think of it, small f, small selection. Big f, big selection. What's done right now makes no sense.

Also, making this an official default would be great. But can a configuration example of this within this issue be provided if possible?

**Comment by pascalkuthe** [comment]
The current default is very much the intended behavior. Helix is workspace centric. We will not change the default.

Swapping the keybdinings in your co fig is easy. Just create two new keybindings for it. It's all documented.
